recipes-kernel/kernel-tests/kernel-containerized-performance-tests-files/test_kernel_cyclictest_fio_containerized.sh

```diff
@@ -37,9 +37,11 @@ RESULT=$(docker run --privileged --network=host \
     bash call_run_ct.sh "fio_containerized" \
         | tr -d '\r' | tr -d '\n')
 
+# Make sure we print the PASS/FAIL message
+cat ${LOG_DIR}/run_cyclictest-`date +'%Y_%m_%d-%H_%M_%S'`-fio_containerized.log
```

Comment:
You're recalculating the timestamp, so if you're unlucky with the clock, these files may not exist.

Reply:
I guess we don't really need the timestamp anyway since the target is wiped every run